packages/data-transformation/raw-to-core/main.py

```
@@ -0,0 +1,51 @@
+    if not user:
+        raise ValueError("Missing required environment variables: USER")
+    os.environ['DBT_PG_USER'] = user
+
+    password = os.getenv("PASSWORD") or os.getenv("DBT_PG_PASSWORD")
+    if not password:
+        raise ValueError("Missing required environment variables: PASSWORD")
+    os.environ['DBT_PG_PASSWORD'] = password
+
+    database = sys.argv[1] or os.getenv("DATABASE") or os.getenv("DBT_PG_DATABASE")
+    if not database:
+        raise ValueError("Missing required environment variables: DATABASE")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)